### DISCUSSION TITLE
Confidence Score for Table Extraction

### DISCUSSION THREAD

**Discussion by parthplc** [jsvine/pdfplumber#769] [Ask for help with specific PDFs]
After the extraction of the table is there any way of getting the confidence score for an extracted table ( describing how accurate the table extraction process could be )?

**Comment by samkit-jain** [comment]
Hi @parthplc the library does not support this at the moment. But this is a good request and shall be considered. If you have suggestions to implement, you are also encouraged to raise a PR solving the same.
